apps/mobile/src/components/AppLogsScreen/index.tsx

```diff
@@ -12,19 +12,21 @@ import {
 import LogsList from './components/LogsList'
 import Button from '../Button'
 import saveLogsToDirectoryAndShare from './utils/saveLogsToDirectory'
-import {useSetAtom} from 'jotai'
+import {useSetAtom, useAtomValue} from 'jotai'
 import {loadingOverlayDisplayedAtom} from '../LoadingOverlayProvider'
 import {Alert} from 'react-native'
 import {setupAppLogs} from './utils/setupAppLogs'
 import IconButton from '../IconButton'
 import closeSvg from '../images/closeSvg'
 import useSafeGoBack from '../../utils/useSafeGoBack'
+import { appLogAtomsAtom } from './atoms'
 
 function AppLogsScreen(): JSX.Element {
   const {t} = useTranslation()
   const [enabled, setEnabled] = useState(getCustomLoggingEnabled())
   const setLoading = useSetAtom(loadingOverlayDisplayedAtom)
   const safeGoBack = useSafeGoBack()
+  const logsAtoms = useAtomValue(appLogAtomsAtom)
```

Comment:
This will cause unnecessary rerender of this component when new logs are written. It would be much better to use a selectAtom like: 

```
const appLogsEmptyAtom = selectAtom(appLogsAtom, logs => longs.length === 0)
```

Reply:
9f3bf36bdb36ae3eb8d540313107c9da459dd4bc